pkg/web/src/app/queries/providers.ts

```diff
@@ -398,13 +407,14 @@ const getCertificate = async (hostname: string) => {
 };
 
 export const useCertificateQuery = (
+  providerType: ProviderType | null,
   hostname: string,
   enabled: boolean
 ): UseQueryResult<ITLSCertificate> => {
   const result = useMockableQuery<ITLSCertificate>(
     {
       queryKey: ['certificate', hostname],
-      queryFn: () => getCertificate(hostname),
+      queryFn: () => getCertificate(providerType, hostname),
```

Comment:
One other thing I notice here: Because you're now using `useCertificateQuery` twice in the above component, we should make sure the underlying `queryKey` being used is unique for those 2 calls or else they'll clobber each other's cache. You can add as much unique identifying info to this `queryKey` array as you want, we should probably put `providerType` in there now.